src/screens/EventDetailsScreen/component.js

```diff
@@ -0,0 +1,225 @@
+// @flow
+import React from "react";
+import { View, StyleSheet, ScrollView } from "react-native";
+import type { NavigationScreenProp } from "react-navigation";
+import formatDate from "date-fns/format";
+import isSameDay from "date-fns/is_same_day";
+import Header from "./Header";
+import IconItem from "./IconItem";
+import CategoryLabel from "./CategoryLabel";
+import EventMap from "./EventMap";
+import Text from "../../components/Text";
+import Button from "../../components/Button";
+import {
+  eventDetailsBgColor,
+  eventDetailsHeaderBgColor
+} from "../../constants/colors";
+import text from "../../constants/text";
+import strings from "../../constants/strings";
+import type { Event } from "../../integrations/cms";
+
+const locale = "en-GB";
+
+type Props = {
+  navigation: NavigationScreenProp<{ params: { eventId: String } }>,
+  event: Event
+};
+
+const renderEventOverview = event => {
+  const startTime = new Date(event.fields.startTime[locale]);
+  const endTime = new Date(event.fields.endTime[locale]);
+  const dateFormat = "DD MMMM YYYY";
+  const timeFormat = "HH:mm";
+  const dateDisplay = isSameDay(startTime, endTime)
```

Comment:
How does this work when the startTime and endTime are not on the same day? Example:

* Start: 2018-06-10 10:00
* End: 2018-06-12 18:00

In this case we would render: 2018-06-10 - 2018-06-12, 10:00 - 18:00.

This sounds like the event happens on three days (10, 11, 12) each from 10:00 to 18:00.

Is this how it is supposed to work?

Reply:
I'm actually going to clarify this with UX because I think this current way doesn't make much sense.

I think you're assumption above is actually how it was intended to work, but I think there's many cases where it wouldn't make sense.

I think Steve has already changed this from the UX side so I'll clarify with him.